django/forms/utils.py

```diff
@@ -114,10 +116,19 @@ def as_ul(self):
         if not self.data:
             return ''
 
+        li = format_html_join('', '<li>{}</li>', ((e,) for e in self))
+        if self.field:
+            return format_html(
+                '<div id="{}_{}"><ul class="{}">{}</ul></div>',
+                self.field.id_for_label,
+                self.error_class,
```

Comment:
Question: Isn’t this going to break if `error_class` is provided in the constructor? I would have expected:

```suggestion
                '<div id="{}_errorlist"><ul class="{}">{}</ul></div>',
                self.field.id_for_label,
```

A few lines up – If I read this right this would lead to having a space in the id, which is not allowed.

```python
self.error_class = 'errorlist {}'.format(error_class)
```

Additionally `_errorlist` seems to be hard-coded as the ID suffix in the `aria-describedby`.